src/main/scala/mesosphere/marathon/MarathonSchedulerService.scala

```diff
@@ -58,6 +58,8 @@ trait DeploymentService {
     * @return a failed future if the deployment failed.
     */
   def deploy(plan: DeploymentPlan, force: Boolean = false): Future[Unit]
+
+  def listRunningDeployments(): Future[Seq[DeploymentStepInfo]]
```

Comment:
We currently use `Future[Seq[T]]` in a lot of places. Please let's defer refactoring in a separate PR.